### ISSUE TITLE
[Functional- Bot Framework Web Chat - Video vimeo]: like control is not functional under more options present in video section.

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5363]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
User will get confused with the functionality of the control and cannot access the control efficiently if the control is not behaving according to the functionality of the control.

### Note:
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Suggested action

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Video vimeo word in type your message text box.
4. Navigate to the search results.
5. Verify the like control is functional or not.

### What do you expect?

like control should be functional under more options present in video section.
### Ex:
using keyboard and mouse comment action should be performed.

### What actually happened?

like control is not functional under more options present in video section.
### Observation:
Using keyboard and mouse no action performed after invoking the like control under more options.
Screen reader (narrator, nvda, jaws) announcing as' like menuitem radio button not selected'

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/13b6e39b-b394-42bc-98ab-e3451475e407
![Image](https://github.com/user-attachments/assets/a81ea4bc-61d8-45cd-9bfd-20c5ef5ed1c2)

**Comment by manojs5** [comment]
#A11ySeV1;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#Functional;#Win11-Edge(Chromium);
